vue/configurator/src/views/MarketplaceView.vue

```diff
@@ -78,6 +78,37 @@ export default class MarketplaceView extends Vue {
                 value: false,
                 defaultValue: false,
               },
+              "data-widget-theme": {
```

Comment:
i'm a bit fuzzy on our previous discussion regarding whether the theming was to affect only connect widget, or all widgets in the preview? the current implementation only affects the connect widget.

Reply:
Sorry! Because the comments I left were pending, it looks like i didn't clarify this data-prop vs widgetdefinition issue. In this comment https://github.com/tingting-lwy/manifold-templates/pull/1#discussion_r1163163219 I think I realize we are on the same page about WidgetDefinition! 👍